cloudbuild.yaml

```diff
@@ -33,7 +33,7 @@ steps:
     path: /root/.ssh
 
 # Run the build
-- name: l.gcr.io/google/bazel:3.5.0
+- name: l.gcr.io/google/bazel:4.2.1
```

Comment:
There's no such thing as l.gcr.io/google/bazel:4.2.1

The highest it goes today 3.5. That's why I install it manually below.

Reply:
Yeah, I thought it was just an internal name and discovered I was wrong. Fixed.